app/assets/stylesheets/darkswarm/footer.scss

```diff
@@ -36,6 +36,7 @@ footer {
 
       img {
         margin-top: 36px;
+        width: 120px;
```

Comment:
Isn't `img` a too broad selector? if the logo is all we want to change, I'd rather use `.logo` instead.

Reply:
This is nested and will be `h1 img`. There is only one `h1` with only one `img`. It doesn't have a logo class. We would need to add something like a `header__logo` class. But I think this pull request is fine as it is, not invasive.

Reply:
Now I see this rule is actually nested within `.logo` so nevermind.